src/module/actor/sheet/base.js

```diff
@@ -987,6 +988,24 @@ export class ActorSheetSFRPG extends ActorSheet {
             return;
         }
 
+        const itemData = await Item.fromDropData(parsedDragData);
```

Comment:
Need to check if this method exists in v9 or not. If not, then we'll need to do more version checking before processing.

Reply:
Can confirm it does

Reply:
Unfortunately, this method doesn't work when the item type is of type `ItemCollection`. This is a custom type that we use with the loot sheets. So, you can drag an item to the canvas, but can't drag that item from the loot sheet onto an actor.